packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -223,9 +314,43 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
 
     private renderDirectionToggle(): React.ReactNode {
         return (
-            <button onClick={this.handleUpdateDirection}>
-                {this.state.viewConfig.direction}
-            </button>
+            <div
+                className={get(
+                    this.props,
+                    "managedClasses.explorer_viewerControlContrainer"
+                )}
+            >
+                <Label jssStyleSheet={this.labelStyleOverrides}>RTL</Label>
+                <Toggle
+                    jssStyleSheet={this.toggleStyleOverrides}
+                    inputId={"direction"}
+                    onClick={this.handleUpdateDirection}
+                    selectedMessage={""}
```

Comment:
These should probably have selected / unselected messages - otherwise it seems like we're using these toggles incorrectly and they should just be checkboxes. Thoughts?

Reply:
+1 I think toggle makes more sense in this case than a checkbox, but the messages should have content.

Reply:
@bheston thoughts on this? I know we wanted to use toggles.

Reply:
chatted offline. Making these props optional on `<Toggle/>`